assets/src/edit-story/components/panels/design/link/link.js

```diff
@@ -280,6 +280,10 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
           <Row spaceBetween={false}>
             <StyledMedia
               value={link.icon || ''}
+              cropParams={{
+                width: 96,
+                height: 96,
```

Comment:
Nit: I think this could easily be reduced to 32x32.

On the frontend, the image is currently displayed in 24x24, but 32x32 is suggested going forward.

Reply:
The reason I picked these values as we already have this image size, https://github.com/google/web-stories-wp/blob/71cd89aa6330b58d5239d504570f7e8e03bcb92f/includes/Media/Media.php#L80

Meaning that you do not have to crop anything. I can change it want.

Reply:
What difference does it make whether we already have such an image size or not?

Reply:
Fixed.